src/app/(auth)/(navigationsBarLayout)/problems/page.tsx

```
@@ -181,32 +181,34 @@ const ProblemsList = () => {
                     {categoryCodeOptions.find((item) => item.value === categoryCode)?.label}
                   </span>
                   <Image
                     src="/icons/close/closeWithBorder.svg"
-                    className="cursor-pointer"
+                    className="cursor-pointer hover:opacity-70 transition-opacity"
                     alt="close"
-                    width={16}
-                    height={16}
+                    width={14}
+                    height={14}
                     onClick={() => setCategoryCode('')}
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)